docs/polkadot/implementors-guide/guide.md

```diff
@@ -0,0 +1,342 @@
+3. Fishermen. These are user-operated, permissionless nodes whose goal is to catch out misbehaving validators in exchange for a bounty. Collators and validators can behave as Fishermen too. Fishermen aren't necessary for security, and aren't covered in-depth by this document.
+
+This alludes to a simple pipeline where collators send validators parachain blocks and their requisite PoV to check. Then, validators validate the block using the PoV, signing statements which describe either the positive or negative outcome, and with enough positive statements, the block can be included. If another validator later detects that a validator or group of validators incorrectly signed a statement claiming a block was valid, then those validators will be _slashed_, with the checker receiving a bounty.
+
+However, there is a problem with this formulation. In order for another validator to check the previous group of validators' work after the fact, the PoV must remain _available_ so the other validator can fetch it in order to check the work. The PoVs are expected to be too large to include in the blockchain directly, so we require an alternate _data availability_ scheme which requires validators to prove that the inputs to their work will remain available, and so their work can be checked.
+
+Here is a description of the Inclusion Pipeline: the path a parachain block (or parablock, for short) takes from creation to inclusion:
+1. Validators are selected and assigned to parachains by the Validator Assignment Process.
+1. A collator produces the parachain block, which is known as a parachain candidate or candidate, along with a PoV for the candidate.
+1. The collator forwards the candidate and PoV to validators assigned to the same parachain via the Collation Distribution Process.
+1. The validators assigned to a parachain at a given point in time participate in the Candidate Backing Process to validate candidates that were put forward for validation. Candidates which gather enough signed validity statements from validators are considered "backed" and are called backed candidates. Their backing is the set of signed validity statements.
+1. A relay-chain block author, selected by BABE, can include up to one (1) backed candidate for each parachain to include in the relay-chain block alongside its backing.
+1. Once included, the parachain candidate is considered to be "pending availability". It is not considered fully included until it is proven available.
+1. In the following relay-chain blocks, validators will participate in the Availability Distribution Process to ensure availability of the candidate. Information regarding the availability of the candidate will be included in the subsequent relay-chain blocks.
```

Comment:
```suggestion
1. In the following relay-chain blocks, validators will participate in the Availability Distribution Process (see https://github.com/w3f/research/pull/77 for network related details) to ensure availability of the candidate. Information regarding the availability of the candidate will be included in the subsequent relay-chain blocks.
```

Reply:
I'd suggest to have a separate section which describes the Availability Distribution Process which then links to that PR. The initial description of the Availability Distribution Process will probably be a gossip-based one which only references @infinity0's work as future work.